firefox-ios/Client/Frontend/TrackingProtection/BlockedTrackersHelpers/BlockedTrackersTableView.swift

```diff
@@ -48,7 +48,7 @@ class BlockedTrackersTableView: UITableView,
         var snapshot = NSDiffableDataSourceSnapshot<Int, BlockedTrackerItem>()
         snapshot.appendSections([0])
         snapshot.appendItems(items, toSection: 0)
-        diffableDataSource?.apply(snapshot, animatingDifferences: true)
+        diffableDataSource?.apply(snapshot, animatingDifferences: false)
```

Comment:
curious - why false here? I'm also working with diffable datasource for the homepage rebuild project.

Reply:
https://github.com/user-attachments/assets/2ab07b7e-7c57-4518-8b6d-b32770c25b0c

trying to lower the number of epileptic seizures the users might experience because of how refreshing the number while the URL loads is animating the differences

Reply:
https://github.com/user-attachments/assets/16932b72-6367-4032-983a-3876d86d11d9

maybe an animation could look nice though, but certainly not the one the SDK provides

Reply:
ah thank you for explaining! yes the first video is not great at all 🤣